kiali/kiali
# [Tempo] Query scope fix

### Describe the change

- Fix query scope for Tempo
- Update hack scripts to be able to test Tempo in multicluster

### Steps to test the PR

Single cluster: 

- `minikube start`
- `istio/tempo/install-tempo-env.sh -c kubectl`
- Verify there are traces

Multi cluster (Primary remote): 

- `hack/run-integration-tests.sh --test-suite frontend-primary-remote --setup-only true --tempo true`
- Verify there are traces filtered by cluster
- Add query_scope with valid value and verify there are traces
- Add query_scope with invalida value and verify there are no traces
```

    query_scope: 
      istio.mesh_id: "mesh-hack"
```

### Automation testing

If applicable, explain the case scencarios covered by **unit / integration / e2e** tests created for this PR.

### Issue reference

Fixes #7658 

TODO: 
- Update tracing script for Tempo to send remote traces to the local cluster (Can be done in a follow up PR)
